internal/sdks/sdks.go

```diff
@@ -23,21 +27,33 @@ func ReplaceFlagKey(instructions string, key string) string {
 	return r.Replace(instructions)
 }
 
-// ReplaceSDKKey changes the placeholder flag key in the SDK instructions to the flag key from
-// the user.
-func ReplaceSDKKey(instructions string, key string) string {
+// ReplaceSDKKeys changes the placeholder SDK key/client side ID in the SDK instructions to the key from
+// the default test environment for the user's account.
+func ReplaceSDKKeys(instructions string, sdkKey, clientSideId string) string {
 	r := strings.NewReplacer(
 		"1234567890abcdef",
-		key,
-		"myClientSideID",
-		key,
+		sdkKey,
+		"myClientSideId",
+		clientSideId,
+		// remove remaining values when we add all hardcoded instructions
 		"mobile-key-from-launch-darkly-website",
-		key,
+		sdkKey,
 		"YOUR_SDK_KEY",
-		key,
+		sdkKey,
 		"Your LaunchDarkly SDK key",
-		key,
+		sdkKey,
+		"myClientSideID",
+		clientSideId,
 	)
 
 	return r.Replace(instructions)
 }
+
+// kebabToCamel converts a kebab-case key string into a camelCase key string, used for the React sdk instructions
```

Comment:
from [the docs](https://docs.launchdarkly.com/sdk/client-side/react/react-web?site=launchDarkly#flag-keys-in-the-react-web-sdk):

>However, in JavaScript, accessing keys containing . and - requires the bracket operator. Instead of requiring this, the React Web SDK automatically changes all flag keys to camel case by default. A flag with key dev-flag-test is accessible as flags.devFlagTest.

so we should do that here also